pom.xml

```diff
@@ -149,8 +149,8 @@
         <version.commons-io>2.5</version.commons-io>
         <version.commons-lang>2.6</version.commons-lang>
         <version.io.undertow>2.0.26.Final</version.io.undertow>
-        <version.javax.inject.javax.inject>1</version.javax.inject.javax.inject>
-        <version.javax.json.javax-json-api>1.1.2</version.javax.json.javax-json-api>
+        <version.jakarta.inject.jakarta.inject-api>1.0</version.jakarta.inject.jakarta.inject-api>
```

Comment:
Version of inject looks correct according to https://github.com/jakartaee/specifications/blob/402583a2c835c20732ad713f823edfbd5c6cec25/dependency-injection/1.0/_index.md

Reply:
This is just an FYI